scripts/check-imports.js

```diff
@@ -35,6 +35,7 @@ function check() {
     'packages/workspace/src/command-line/shared.ts',
     'packages/workspace/src/schematics/preset/preset.ts',
     'packages/workspace/src/schematics/init/init.ts',
+    'packages/workspace/src/utils/collections.ts',
```

Comment:
This doesn't need to be here anymore. It should probably be `plugins.ts` if anything.